tacs/constitutive.pyx

```diff
@@ -478,23 +478,42 @@ cdef class MaterialProperties:
         """
         self.ptr.setSpecificHeat(specific_heat)
 
+cdef class MAT2MaterialProperties(MaterialProperties):
+    """This class only exists so that BladeStiffenedShellConstitutive objects can return a TACS MaterialProperties
+    instance that generates the necessary MAT2 nastran cards
+    """
+    def __cinit__(self, E1, G12, G13, E2, G23, E3, rho=0.0):
+        self.ptr = NULL
+        self.E1 = E1
+        self.G12 = G12
+        self.G13 = G13
+        self.E2 = E2
+        self.G23 = G23
+        self.E3 = E3
+        self.rho = rho
+        self.nastranID = 0
+
+    def generateBDFCard(self):
+        return nastran_cards.materials.MAT2(self.nastranID, np.real(self.E1), np.real(self.G12), np.real(self.G13),
+                                            np.real(self.E2), np.real(self.G23), np.real(self.E3), np.real(self.rho))
+
```

Comment:
Do we need to make a new class for this? It seems we could just re-use the `MaterialProperties` class?

Reply:
Nevermind, I see this is because the class needs MAT2 cards specifically, since the stiffnesses are anisotropic